src/model/annotation-list.v1.yaml

```diff
@@ -0,0 +1,62 @@
+$schema: http://json-schema.org/draft-04/schema#
+title: Annotation list
+type: object
+properties:
+    total:
+        title: Total number of Annotations
+        type: integer
+        minimum: 0
+    items:
+        title: Annotations
+        type: array
+        items:
+            type: object
+            properties:
+                id:
+                    description: Annotation ID
+                    type: string
+                private:
+                    description: If the annotation should only be visible to the author
+                    type: boolean
+                    default: false
```

Comment:
We should match what we did for profiles https://github.com/elifesciences/api-raml/blob/d2c08de4c7f3493cb8bbed9200b23f2cab0da226/src/model/profile.v1.yaml#L14-L15